input/resources/au-patient.xml

```diff
@@ -104,20 +111,23 @@
       <path value="Patient.extension" />
       <sliceName value="genderIdentity" />
       <short value="The patient's gender identity shall be a member of the Gender Identity Response value set if any of the codes within that value set can apply"/>
-      <comment value="Gender identity [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) ([Extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). See [Australian Bureau of Statics (ABS) Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) for further materials."/>
+      <comment value="Individual Gender identity [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) ([Extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). See [Australian Bureau of Statics (ABS) Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) for further materials."/>
       <type>
         <code value="Extension" />
-        <profile value="http://hl7.org/fhir/StructureDefinition/patient-genderIdentity" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/individual-genderIdentity" />
       </type>
       <condition value="inv-pat-1"/>
     </element>
     <element id="Patient.extension:individualPronouns">
       <path value="Patient.extension"/>
       <sliceName value="individualPronouns"/>
+      <short value="The patient's pronouns shall be a member of the Australian Pronouns value set if any of the codes within that value set can apply"/>
```

Comment:
Shorten the text. Replace "The patient's pronouns" with "Pronouns"

Reply:
Done

Reply:
Done

Reply:
1. Confirmed fix
2. Not yet fixed